app/Transformers/Multiplayer/PlaylistItemTransformer.php

```diff
@@ -47,7 +48,7 @@ public function includeBeatmap(PlaylistItem $item)
     public function includeDetails(PlaylistItem $item)
     {
         return $this->primitive($item->detailEvent->event_detail ?? [
-            'room_type' => $item->room->type,
+            'room_type' => Room::REALTIME_DEFAULT_TYPE,
```

Comment:
wait, what's this change for?

Reply:
otherwise it may be shown as team match which wouldn't make sense without the team member data.